### ISSUE TITLE
License Roadmap, missing LICENSE file

### ISSUE THREAD

**Issue by gautamsi** [Microsoft/fonts#9]
I see that this is supposed to be open source, what is the plan going forward? I was looking to see some update but this one and Selawik font has not been updated for over one year. 

Is Microsoft still going to support this font and bring it compatible with Segoe font? 

I also see the license file is missing, there is no mention of type of license in README either.

**Comment by aaronbell** [comment]
Yes, the plan is to continue to update this font. However, there have been other higher priority items that have been taking up my time so work on this font has paused for the time being. I am looking to be making updates to the font in the coming month or two, but cannot commit to any specific improvements or roadmap. If there are any items that you would like to see improved upon, please feel free to file an issue and I will consider everything in my next round of updates. 

Thanks!

**Comment by jolleekin** [comment]
Any update so far?

**Comment by riverar** [comment]
Ping @aaronbell

**Comment by aaronbell** [comment]
The ball is in Microsoft’s court. While I’d like to make updates to the font, I don’t have any spare time to work on it. So right now it is up to them to want to invest in it.
